app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
> A post with reproduction instructions and what your expectations are would be great 

My expectations are that I would be able to see the stack trace when I get an exception in development. This was not clear from the above?